### PR TITLE
check for CATKIN_ENABLE_TESTING

### PR DESCRIPTION
**Pull request by bulwahn**

### CODE REVIEW

File: joint_trajectory_controller/package.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
   <author>Adolfo Rodriguez Tsouroukdissian</author>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend version_gte="0.5.68">catkin</buildtool_depend>
```
Comment (davetcoleman) [inline joint_trajectory_controller/package.xml:16]: I know it means:

> The dependency to the package is restricted to versions greater or equal than the stated version number.

But why is this necessary and should we have it in all the ros_controllers packages?

Comment (bulwahn) [inline joint_trajectory_controller/package.xml:16]: The variable CATKIN_ENABLE_TESTING is only available since catkin 0.5.68. To have the intended behavior for CATKIN_ENABLE_TESTING=ON and OFF, you need catkin 0.5.68 or later. It is okay to have this constraint only in the packages that use this feature. Most people on hydro have 0.5.68 installed anyway.
